app/components/candidate_interface/new_references_review_component.rb

```diff
@@ -183,11 +183,12 @@ def reference_type_row(reference)
     def status_row(reference)
       return nil unless reference.feedback_provided?
 
-      double_break = tag.br + tag.br
-
       {
-        key: 'Status',
-        value: feedback_status_label(reference) + double_break + t('application_form.new_references.status.first_line', name: reference.name) + double_break + t('application_form.new_references.status.second_line'),
+        key: '',
+        value: [
```

Comment:
@frankieroberto this is providing to be a bit awkward to get the right sized gap in between the two lines. It's either like this or too big:
![image](https://user-images.githubusercontent.com/47917431/188391642-95b1bb73-dbed-4cc2-a6b7-376da177c58d.png)

Preference 😬 ?

Reply:
This one (with the space inbetween) looks good to me - it should ideally be output using regular paragraphs:

 ```html
 <p class="govuk-body">... has already given a reference.</p>
 
 <p class="govuk-body">If you accept an offer, the training provider will see the reference.</p>
 
 ```

Reply:
👍